examples/service-catalog/service-catalog.yaml

```diff
@@ -0,0 +1,411 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: service-catalog
+objects:
+
+- kind: ServiceAccount
```

Comment:
i figured we might as well use the default SA for *something* but yeah, i can rework it.